<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecursoForm</class>
 
 <!-- Ventana con centralizado -->
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="main_layout">


    <!-- Fila 1: Id -->
    <item>
    <layout class="QGridLayout" name="gridLayout">
        
     <item row="0" column="0">
      <widget class="QLabel" name="label_id">
       <property name="text">
        <string>Id</string>
       </property>
       
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>

      </widget>
     </item>
     
     <item row="0" column="1">
      <widget class="QLineEdit" name="entry_id"/>
     </item>

     <!-- Fila 2: Nombre Apellido Paterno y Materno -->        
     <item row="1" column="0">
      <widget class="QLabel" name="label_name">
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
     </item>
     
     <item row="1" column="1">
      <widget class="QLineEdit" name="entry_name"/>
     </item>
     
     <!-- Apellido paterno -->
     <item row="1" column="2">
      <widget class="QLabel" name="label_paternal_surname">
       <property name="text">
        <string>Apellido paterno</string>
       </property>
      </widget>
     </item>
     
     <item row="1" column="3">
      <widget class="QLineEdit" name="entry_paternal_surname"/>
     </item>
     
     <!-- Apellido materno -->
     <item row="1" column="4">
      <widget class="QLabel" name="label_maternal_surname">
       <property name="text">
        <string>Apellido materno</string>
       </property>
      </widget>
     </item>
     
     <item row="1" column="5">
      <widget class="QLineEdit" name="entry_maternal_surname"/>
     </item>
    
    
     <!-- Fila 3: Puesto -->
     <item row="2" column="0">

      <widget class="QLabel" name="label_position">

       <property name="text">
        <string>Puesto</string>
       </property>

      </widget>

     </item>
     
     <item row="2" column="1">
      <widget class="QLineEdit" name="entry_position">

      </widget>
     </item>
    



     <!-- Fila 4: Baja Checkbox -->
     <item row="3" column="0">
      <widget class="QLabel" name="label_soft_delete">
       <property name="text">
        <string>Baja</string>
       </property>
      </widget>
     </item>
     
     <item row="3" column="1">
      <widget class="QCheckBox" name="checkbox_soft_delete">
      </widget>
     </item>

    </layout>
    </item>
    
    
    
    
    <!-- Insertar y actualizar -->
    <item>
    <layout class="QHBoxLayout">
        
     <item>
      <widget class="QPushButton" name="button_add">
       <property name="text">
        <string>Agregar</string>
       </property>
      </widget>
     </item>
     
     <item>
      <widget class="QPushButton" name="button_update">
       <property name="text">
        <string>Actualizar</string>
       </property>
      </widget>
     </item>
     
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text"><string>Cancel</string></property>
      </widget>
     </item>

    </layout>
    </item>
    
    
    
    
    <!-- Tabla -->
    <item>
     <widget class="QTableWidget" name="table">
     </widget>
    </item>
    
   </layout>
  </widget> <!-- Central widget para centrar todo -->

 <resources/>
 <connections/>
</ui>